src/providers.ts

```diff
@@ -31,6 +32,13 @@ const LD_MODE: DocumentFilter = {
 };
 
 export function register(ctx: ExtensionContext, config: Configuration, flagStore: FlagStore, api: LaunchDarklyAPI) {
+	const flagView = new ldFeatureFlagsProvider(api, config, flagStore, ctx);
+	if (getIsTreeviewEnabled()) {
+		commands.executeCommand('setContext', 'launchdarkly:enableTreeview', true);
+	}
```

Comment:
I was able to comment out this block and still have the tree view automatically rerender when the boolean option is changed - maybe it's not necessary?

Reply:
This option is for completely disabling the treeview provider but leaving the rest of the plugin functionality still usable.